src/hooks/useTokenMaps.ts

```diff
@@ -0,0 +1,43 @@
+import { BasicTokensMap, PoolName, TokenToPoolsMap } from "../constants"
+
+import { BasicPoolsContext } from "../providers/BasicPoolsProvider"
+import { TokensContext } from "../providers/TokensProvider"
+import { useContext } from "react"
+
+export const useTokenMaps = (): {
```

Comment:
Should this also account for `underlyingTokens`, or just `tokens`?

Reply:
THis is a good catch. I believe it should be for underlyingTokens since they switched since bringing in PoolsRegistry.